### ISSUE TITLE
apim-cli.sh not working when JAVA_HOME doesn't exist

### ISSUE THREAD

**Issue by lacg** [Axway-API-Management-Plus/apim-cli#64]
**API-Mgr. CLI version**
Version 1.2.1
> Please make sure to use the latest available version before reporting a bug.

**API-Manager and Service-Pack Version**
N/A

**Expected behavior**
Run the shell script with the command usage if you type ./apim-cli.sh

**Actual behavior**
Error: Could not find or load main class com.axway.apim.cli.APIManagerCLI

**Log-Output in Debug**

**Test case sample**
> Please describe the steps you have done to replicate the issue

I fixed it by adding/changing the lines after programDir variable declaration:
baseClassDir="$programDir/.."
cd $baseClassDir

CP=$baseClassDir/lib:$baseClassDir/conf
for jars in lib/*
do
        CP=$CP:$baseClassDir/$jars
done

This case, the classes are not relative anymore, but absolute, thus not having issues to find the main class any longer. Feel free to change to whatever you like as well, as you can have other limitations/constraints/standards in mind.

Thanks, Luiz

**Comment by cwiechmann** [comment]
Hi @lacg, Thanks for the feedback! 

I assume you have **not** downloaded the latest release, but instead cloned the project and called the script like so:
```
[axway@api-env apim-cli]$ ./modules/apim-cli/assembly/scripts/apim.sh
/usr/bin/java

Error: Could not find or load main class com.axway.apim.cli.APIManagerCLI
```

Please download the latest release package from here: https://github.com/Axway-API-Management-Plus/apim-cli/releases I have tested this with a fresh installation and it's working as expected.

**Comment by lacg** [comment]
Hi @cwiechmann, I downloaded from [here](https://github.com/Axway-API-Management-Plus/apim-cli/releases/tag/apimcli-1.2.1) and it says latest release. Though, I can double check tomorrow morning...
Thanks, Luiz

**Comment by cwiechmann** [comment]
Hmm, perhaps you have downloaded the source-release-package instead of the distribution package. I'm waiting for your feedback.

**Comment by lacg** [comment]
Yeah, I downloaded the 1.2.1 latest release!
I had cloned earlier on, but I couldn't make it work and I realized it was not the right way (and the folder was not easy to find (/modules/apim-cli/assembly/scripts/). Thus I used the tar file for the Mac and the Zip file for the Windows and both have the 1.2.1 name on the folder with /scripts on the subfolder, so it is not the wrong version and the wrong script.
Don't get me wrong, if JAVA_HOME is well defined, it should work just fine and if it was not it was supposed to work and I have no idea why it is not on Windows (on Linux/Mac, it will depend on how the OS treats the relative path/working directory). I had issues before with certain versions of bash/zsh not changing the working directory with cd .. (most does though)
To solve this, or we could change the pwd variable manually (there is no perfect way on doing this 100% of the time on all shells) or use absolute paths (a problem with longer paths and class paths character limits). In the end, no solution will be perfect... :( As the jar files are not that much, I went with the absolute path... but that is no way THE only solution.
